tfe/resource_tfe_organization.go

```diff
@@ -0,0 +1,153 @@
+package tfe
+
+import (
+	"fmt"
+	"log"
+
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+)
+
+func resourceTFEOrganization() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceTFEOrganizationCreate,
+		Read:   resourceTFEOrganizationRead,
+		Update: resourceTFEOrganizationUpdate,
+		Delete: resourceTFEOrganizationDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"email": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"session_timeout": &schema.Schema{
```

Comment:
nit: I generally like to include a unit with time values, to avoid ambiguity and leave options open in the future.

Reply:
We can update that. Currently this follows the behavior of the API, but I'm fine with updating it. Do we have a standard lib for converting (e.g. `1h` to `60m`) back and forward? If so (which I'm guessing :wink:) this should be a simple small tweak...

Reply:
As I mentioned in DM, updating here for posterity, I just meant in the field name, not adding `time.Duration` string format support to the value.

Reply:
Fixed...